File: src/__fixtures__/sample-py/services/user_service.py
# Sample Python module for testing
from typing import Optional
from .models import User
from ..shared.database import Database
import logging

class UserService:
    """User service for CRUD operations"""
    
    def __init__(self, db: Database):
        self.db = db
        self.logger = logging.getLogger(__name__)
    
    def get_user(self, user_id: str) -> Optional[User]:
        return self.db.find_one("users", {"id": user_id})
    
    def create_user(self, data: dict) -> User:
        self.logger.info(f"Creating user: {data}")
        return self.db.insert("users", data)

File: src/__fixtures__/sample-ts/services/userService.ts
// Sample service with multiple imports
import { DatabaseClient } from '../shared/database';
import { Logger } from '../shared/logger';
import { cache } from '../utils/cache';
import type { User, UserCreateInput } from '../types';

export class UserService {
  constructor(
    private db: DatabaseClient,
    private logger: Logger = new Logger('UserService')
  ) {}

  async findById(id: string): Promise<User | null> {
    const cached = cache.get(`user:${id}`);
    if (cached) return cached as User;

    const user = await this.db.users.findUnique({ where: { id } });
    if (user) cache.set(`user:${id}`, user);
    return user;
  }

  async create(input: UserCreateInput): Promise<User> {
    this.logger.info('Creating user', input);
    return this.db.users.create({ data: input });
  }
}
